configs/patternfly.json

```diff
@@ -13,30 +13,317 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "header nav [aria-current]",
+      "selector": "header nav li a",
```

Comment:
Why did you remove the `[aria-current]`selector? Globals selectors should only match one element from the page, the active one.

Reply:
Thank you, I will revert this or at least ensure it targets just the active one